### DISCUSSION TITLE
personal-development-dey-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#31] [Announcements]
# personal-development-dey-1402/

به سبک و سیاقی که الان یک ساله ادامه پیدا کرده، گزارش دی ماه ۱۴۰۲ رو منتشر می‌کنم.
نسیم طالب هنوز درگیر کتاب نسیم طالب (تو کتاب فریب خورده تصادف) هستم. تازه دارم می‌فهمم چی میگه این بشر. عملاً داره یه سری مباحث آماری مثل امید ریاضی و چولگی و احتمال بیشینه رو به زبان ساده و با مثال های متعدد توضیح میده.
ولی خیلی زیاد حرف می‌زنه. حوصله‌ام سر میره بعضاً.

https://aprd.ir/personal-development-dey-1402/

**Comment by hiwasham** [comment]
سلام امیر جان
پست خوبی بود چند تا سرنخ هم برای خودم ذخیره کردم.
منم ده پونزده سال تهران بودم و این مشکلی که میگی رو داشتم باهاش
یه بار هم یه چیزی در موردش نوشتم که البته بیشتر از سر کلافگی لحظه ای بود اما شعر گروس که گذاشته بودم شاید برات جالب باشه:
http://hiwasham.com/%D8%AA%D9%87%D8%B1%D8%A7%D9%86-%DA%A9%D9%84%D8%A7%D9%87-%D8%A8%D8%B2%D8%B1%DA%AF%DB%8C-%D8%A7%D8%B3%D8%AA/
الان به دلایلی (که به قول محمدرضا مربوط به اغتشاشات پارسال مسئولین هست) تبریز هستم تا حداقل دو سال و خیلی راضی هستم. چالشهایی که تهران داشتم رو اینجا اصلا ندارم و سبک و کیفیت زندگیم بهتر شده.
در مورد سریال هم یه ایده به ذهنم رسید. میتونی سریالهایی رو نگاه کنی که هنوز کامل منتشر نشدن. در واقع با همون سرعتی که مثلاً نتفلیکس منتشر میکنه سریال رو نگاه میکنی. 
قربانت

**Reply by pourmand1376** [reply]
سلام هیوا جان. 

جالبه که شما هم تجربه مشابهی از زندگی در تهران دارید. البته مطمئن نیست که دائم بتونم دورکار تو یزد کار کنم و ممکنه بعد از مدتی دوباره مجبور بشم بیام تهران. ولی خب حداقل یکی دو سالی هستیم.

خیلی جالبه که دیدم آپاراتمان‌های تهران را به قوطی کبریت تشبیه کردید. منم دقیقاً همین مورد تو ذهنم بود. برای منی که تو خونه کوچک‌تر از ۲۰۰ ۳۰۰ متر زندگی نکردم، خونه‌های ۵۰ ۶۰ متری تهران قوطی کبریته.

یزد برای من نماد آرامشه. یه زندگی آروم و عمیق. فقط امیدوارم یزدی که تو ذهنم هست با یزدی که الان واقعاً هست، یکی باشه و حداقل نزدیک باشه. 

راجع به سریال هم باز اینطوری که می‌گید ذهنم درگیر میشه و یه هفته به قسمت بعد فکر می‌کنم که حالا چی میشه (می‌دونم مشکل از ذهن مریض منه). دوست دارم ذهنم درگیر کارم بشه و سریال هم همین‌طوری باشه. شاید هم بشه سریالی دید که ذهن رو درگیر نکنه ولی فکر نکنم دیدن اون سریال‌ها ارزش چندانی داشته باشه :)

**Comment by yjavaherian** [comment]
سلام. امروز باید برای تکلیف درس تئوری اطلاعات یه مقاله رو میخوندم و به زبان خودم می نوشتم. نوشتن خلاصه که تموم شد به کلم زد یه کاری که خیلی وقته میخوام انجام بدم ولی انجام نمیدم ( ساختن بلاگ شخصی) رو به بهونه پست کردن خلاصه مطالعه ام انجام بدم. اینه که گشت و گزار برای پیدا کردن یه بستر خوب و یه تم خوب رو شروع کردم. که آخر ماجرا ختم شد  به jekyll و github pages و al-folio. بعد داشتم تو ریپو ai-folio تاب میخوردم چشمم افتاد به شما. گفتم که من این قیافه رو میشناسم ( از گروه مستر بچه های شریف تو تلگرام) و اینجوری شد که با بلاگ شما همین الان آشنا شدم. سلسله اتفاقاتی که منجر شد من این پست شما رو بخونم برام خیلی جالب بود برای همین گفتم که باتون به اشتراک بگذارم.
خداقوت 👋

**Reply by pourmand1376** [reply]
الان رفتم دیدم که تو چند تا گروه مشترک عضو هستیم ولی فکر کنم تا حالا شما رو ندیدم یا اگر دیدم هم اسم‌تون به چهره Map نشده برام. البته این که تلگرام‌تون عکس نداره هم بی‌تاثیر نیست. :) 

این وبلاگی هم که می‌بینید کلبه کوچیکی هست که برای خودم درست کردم و بعضی وقت‌ها یه چیزهایی که بهشون فکر می‌کنم رو اینجا می‌گذارم. عملاً یه فضایی هست که به دور از هیاهوی تلگرام و ایسنتا و بقیه شبکه‌های اجتماعی برای خودم درست کردم و می‌شینم داخلش حرف می‌زنم. خلاصه خوبه. راضیم ازش.
